src/include/daos/object.h

```diff
@@ -431,6 +431,7 @@ enum daos_recx_type {
 struct daos_recx_ep {
 	daos_recx_t	re_recx;
 	daos_epoch_t	re_ep;
+	uint32_t	re_rec_size;
```

Comment:
just a note that we may need to refine it later to avoid add RPC size too much, as daos_recx_ep maybe used for iom later.